lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -15,6 +15,10 @@ $nx-filter-input-height: 20px;
   display: inline-block;
   vertical-align: middle;
   white-space: nowrap;
+  
+  & > * {
+    box-sizing: content-box !important;
```

Comment:
Needed the !important here to ensure that the correct box-sizing overrides a different box-sizing regardless of selector specificity.

Reply:
> Needed the !important here to ensure that the correct box-sizing overrides a different box-sizing regardless of selector specificity.

I'm not sure we want to go this far.  We really, really don't like `!important`

Reply:
> Without important, the example that has a globally set `box-sizing: border-box` doesn't work correctly. This should be safe to do here since NxFilterInput cannot have children other than the ones it renders.

Yeah, I'm with Ross on this, we should never use `!important`. How exactly is the global style in NXRM implemented?

Reply:
To elaborate, IMO adding the box-sizing attribute to this element makes sense because it's necessary for the element to display correctly (and there's a small amount of debate about even that). Adding `!important` though isn't necessary for the component to display correctly everywhere, it's necessary to get the component to display correctly *in a specific product*. In such a case the override belongs in that specific product, not in RSC.

Reply:
> Without important, the example that has a globally set box-sizing: border-box doesn't work correctly. This should be safe to do here since NxFilterInput cannot have children other than the ones it renders.

I'd be in favor of specifying the `box-sizing` explicitly on the elements that need it, and using the parent class as well to get a higher specificity.  For example `.nx-filter-input .nx-filter-input__addon` beats `.x-body *`

Reply:
That makes more sense and should alleviate the need for `!important`. The style in nxrm is basically the exact same `.someclass * { box-sizing: border-box }`.

Reply:
Ok, I made another change. I moved the child classes (`.nx-filter-input__add-on`, etc) inside the parent class `.nx-filter-input` and put the two necessary box-sizing declarations on the appropriate child classes. This makes them more-specific than the global override so they apply without the need for `!important`.

Reply:
> That makes more sense and should alleviate the need for `!important`. The style in nxrm is basically the exact same `.someclass * { box-sizing: border-box }`.

This is a better solution than !important